ui.tests/test-module/specs/modal/modal.runtime.spec.js

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
```

Comment:
Please add an authoring test also to add modal component and basic configuration !